kolibri/core/content/serializers.py

```diff
@@ -34,6 +34,7 @@ class ChannelMetadataSerializer(serializers.ModelSerializer):
     lang_name = serializers.SerializerMethodField()
     available = serializers.SerializerMethodField()
     num_coach_contents = serializers.IntegerField(source="root.num_coach_contents")
+    is_leaf = serializers.SerializerMethodField()
```

Comment:
Don't include `is_leaf` on serializers for `ChannelMetadata` or anything that's not a ContentNode. On the frontend, we may use the `ContentCard` for lots of non-ContentNode things (which we should probably move away from), but we know channels and exams, etc. are not leaf nodes already.